uktrade/data-hub-frontend
# feature/RR-849-get-export-redux-actions

## Description of change

Added the redux actions and state tasks to get an export item by id

## Test instructions

Using redux dev tools, dispatch the following event. Checking the chrome network tab, you should see an XHR request to `api-proxy/v4/export/402b4745-96fb-4c9f-aa07-111e09faa8dd` with a JSON response that contains the export that was requested

```
{
  payload: '402b4745-96fb-4c9f-aa07-111e09faa8dd',
  onSuccessDispatch: 'EXPORT_LOADED',
  type: 'TASK__START',
  id: 'exportDetails',
  name: 'TASK_GET_EXPORT_DETAIL'
}
```



## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
